<!-- -*- mode: XML -*- -->
<!-- run vslam for localization -->

<launch>

  <!-- declare arguments with default values -->

  <!-- node for vslam 192.168.196.110-->
  <node pkg="calmcar" type="calmcar_pub_roi" name="kbd_cam_roi_node" output="screen" >
    <param name="target_ip" value="192.168.196.10" />
    <param name="port" value="9000" />
  </node>

</launch>
